set.mm

```diff
@@ -522922,6 +522938,70 @@ Real and complex numbers (cont.)
     UAEZVKBTQEZEZVLBTNDZEZEZVNBCFVPVMEZVQVNEVOQEVRNPGVOQTHIVPVMBHIIVMUAEZVNVLEU
     MVSUNQRTHIVMUABHIIUGVLVKEUOUARBHIIBRKIURRAHIIARKII $.
 
+  $( Distribute restricted quantification over a biconditional.  (Contributed
+     by Scott Fenton, 7-Aug-2024.) $)
+  rexbi $p |- ( A. x e. A ( ph <-> ps ) ->
+  ( E. x e. A ph <-> E. x e. A ps ) ) $=
+    ( wb wral cv wcel wa wex wrex wal wi df-ral pm5.32 biimpi alimi exbi df-rex
+    sylbi syl 3bitr4g ) ABEZCDFZCGDHZAIZCJZUEBIZCJZACDKBCDKUDUFUHEZCLZUGUIEUDUE
+    UCMZCLUKUCCDNULUJCULUJUEABOPQTUFUHCRUAACDSBCDSUB $.
+
+  ${
+    $d F x $.  $d A x $.  $d B x $.
+    $( The image of a function under a singleton.  (Contributed by Scott
+       Fenton, 8-Aug-2024.) $)
+    fvimasn $p |- ( ( F Fn A /\ B e. A ) -> ( F " { B } ) = { ( F ` B ) } ) $=
+      ( vx wfn wcel wa csn cima cv wbr cab wceq imasng adantl velsn eqcom bitri
+      cfv fnbrfvb syl5rbb abbi1dv eqtrd ) CAEZBAFZGZCBHIZBDJZCKZDLZBCSZHZUEUGUJ
+      MUDDBACNOUFUIDULUHULFZUKUHMZUFUIUMUHUKMUNDUKPUHUKQRABUHCTUAUBUC $.
+  $}
```

Comment:
It appears this is the same as [`~fnsnfv`](https://us.metamath.org/mpeuni/fnsnfv.html), just commuted.

Reply:
Ok, theorem is removed